common/helper.py

```diff
@@ -265,6 +265,9 @@ def extract_archive(archive_path, extract_to):
 
     :param extract_to: Path to extraction
     :type extract_to: String|pathlib.Path
+
+    :param exclude: Patterns for files and directories that should not be extracted
```

Comment:
should be specified what the patterns are. In this case it is just path sub-strings with no wildcards or regexp